src/channelnumber.js

```diff
@@ -0,0 +1,6 @@
+let ChannelNumber = {
```

Comment:
channelnumber.js is added in root folder of the project, please move it to proper location eg create separate component.

Reply:
channelnumber.js moved into NumberInput folder